### PR TITLE
implement keyboard selection

### PR DESCRIPTION
**Pull request by mwhudson**
Add a view immediately after language selection. Please review/try out!

### CODE REVIEW

**Comment by mwhudson** [comment]
I've pushed up the changes I made to make the layouts translatable. It's actually a bit simpler than the previous version I think.

**Comment by mwhudson** [comment]
[Subiquity: keyboard / locales / etc](https://trello.com/c/XIkUwfuJ/192-subiquity-keyboard-locales-etc)

File: subiquity/models/keyboard.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+    def load_language(self, code):
+        if code == self.current_lang:
+            return
+
+        self._clear()
+
+        raw = gzip.open(self._kbnames_file)
+        try:
+            with io.TextIOWrapper(raw) as kbdnames:
```
Comment (cyphermox) [inline subiquity/models/keyboard.py]: Does kbdnames need to be compressed? It would be on a CD, in a squashfs that is already compressed, etc.

**Comment by cyphermox** [comment]
Looks good to me, I tested the keyboard code in a subiquity image and although the install failed in the end on hardware, the keyboard interface looked great and seemed to work as intended. I also did code review and didn't see anything horrible jump out aside from the small details in inline comments.

**Pull request merged**